elasticsearch.md

```diff
@@ -920,6 +920,46 @@ After adding the `filter` rule \(using the block duplication strategy\).
       kibana_index: ".kibana_@{user}"
 ```
 
+#### `response_fields`
+
+This rule allows filtering Elasticsearch responses using defined list of fields. It works in very similar way to fields rule. However it **doesn't make use of Field Level Security \(FLS\)** and could be applied to every response returned by Elasticsearch.
```

Comment:
maybe we should mention that in contrast to `fields` rule which filters out document fields, this rule filters our response fields?